src/components/SideMenu.vue

```diff
@@ -285,10 +285,16 @@ export default defineComponent({
         {
           href: `/${this.selectedBranch?.id}/account`,
           title: 'Account',
+          disabled: !this.userStore.checkAccessLevel(
+            [this.userStore.accessLevels.reblazeOrganizationAdmin, this.userStore.accessLevels.reblazeAdmin],
+          ) && this.userStore.getCookieProfileProvider() === 'reblaze',
```

Comment:
```suggestion
          disabled: !this.userStore.checkAccessLevel(
            [this.userStore.accessLevels.reblazeOrganizationAdmin, this.userStore.accessLevels.reblazeAdmin],
          ) || this.userStore.getCookieProfileProvider() !== 'reblaze',
```

The page is disabled if the user doesn't have enough access
```
!this.userStore.checkAccessLevel(
            [this.userStore.accessLevels.reblazeOrganizationAdmin, this.userStore.accessLevels.reblazeAdmin],
          )
```
OR
isn't connected through our provider
```
this.userStore.getCookieProfileProvider() !== 'reblaze'
```

Reply:
Right, done.